### DISCUSSION TITLE
怎么提交时只提交表单中存在的值

### DISCUSSION THREAD

**Discussion by RiverLoveMU** [alibaba/formily#3601] [Q&A 2.x] [answered]
举个例子：
表单中存在三个输入框{a: string, b: string, c: string}
但是我从后端拿到数据时，后端数据为{a: string, b: string, c: string, d: string, e: string}
然后我将这些数据放入initvalues中，再编辑这些数据
最后提交时得到的数据还是为{a: string, b: string, c: string, d: string, e: string}
但是我实际的需求是提交表单中的值{a: string, b: string, c: string}
有什么办法可以判断我的初始值是否存在表单中吗或者提交时只提交表单中存在的值

**Comment by janryWang** [comment, marked as answer]
自己手动过滤
